Mobile/starter_project_matador/lib/features/example/presentation/screen/onboarding_1_screen.dart

```diff
@@ -0,0 +1,73 @@
+import 'package:flutter/material.dart';
+import 'package:matador/features/example/presentation/widgets/animated_entry.dart';
+import 'package:matador/features/example/presentation/widgets/introduction.dart';
+
+class OnBoarding1 extends StatefulWidget {
```

Comment:
Changing the name to something like OnboardingScreen1 would be better

Reply:
is there maybe a better name to give than onborading screen 1?(1 as compared to what?) idk about the project but the name doesn't describe it well to me so far so I am asking

Reply:
> Changing the name to something like OnboardingScreen1 would be better

Ok, I will do that.

Reply:
> is there maybe a better name to give than onborading screen 1?(1 as compared to what?) idk about the project but the name doesn't describe it well to me so far so I am asking

There is a second onboarding page that I am currently working on and that is why it is named like that.